src/pages/CookieClicker.vue

```diff
@@ -14,22 +15,29 @@ setInterval(() => {
 
 function cookieClick(){
     cookies.value += 1;
+    checkAchievements();
+}
+
+function checkAchievements() {
+    if (cookies.value === 10 && !achievements.value.includes("10 Clicks Milestone!")) {
```

Comment:
Cool this is one but if you need to neatly describe hundreds this is kinda meh. Put some tought into how to make it nice and easy to scale up and make it easy to add lots and lots of achievments with least ammount of code.